packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -54,43 +60,45 @@ git clone git@github.com:department-of-veterans-affairs/vets-api.git
 git clone git@github.com:department-of-veterans-affairs/vets-api-mockdata.git
 ```
 
-These repos are the most important to have a basic understanding of:
-
-- vets-website
-  - Front end platform and application code
-- vagov-content
-  - Markdown content used to generate static pages (must be a sibling of vets-website)
-- vets-json-schema
-  - Shared JSON Schema definitions used by form applications and the APIs that they consume
-- veteran-facing-services-tools
-  - Shared code that we support for the broader VA (not just VA.gov), plus this documentation site
-- vets-api
-  - The repository for the Rails API backend.
-- vets-api-mockdata
-  - Mock data used when running locally and on dev for the backend (must be a sibling of vets-api)
+*Front end repos*
+- **vets-website**: Core front end platform and application code
+- **vagov-content**: Markdown content used to generate static pages
+- **veteran-facing-services-tools**: Shared front end components (including non VA.gov users) and front end documentation site
 
-## Start up the front end
+*Back end repos*
+- **vets-api**: Core Rails API server application code
+- **vets-api-mockdata**: Mock data used when running locally and on dev for the backend
 
-Once you have Node and Yarn installed, you can install the dependencies for vets-website:
+*Shared repos*
+- **vets-json-schema**: Shared JSON Schema definitions used by form applications and the APIs that they consume
 
+## Start up the front end
+
+7. Install `vets-website` dependencies
 ```bash
 yarn install
 ```
 
-Once this completes, you should be able to start up the site locally:
-
+8. Start the local development server
 ```bash
 yarn watch
 ```
 
-Once this builds (you'll see a message about Webpack compilation finishing), open up [http://localhost:3001](http://localhost:3001) and you should see the VA.gov homepage.
+*Build is complete when the CLI says* `Compiled successfully`
```

Comment:
We should try to be as explicit as possible when setting expectations about feedback.